static/js/components/ReplyButton_test.js

```diff
@@ -0,0 +1,77 @@
+// @flow
+import React from "react"
+import { mount } from "enzyme"
+import { assert } from "chai"
+
+import ReplyButton from "./ReplyButton"
+import LoginPopup from "./LoginPopup"
+import Router from "../Router"
+import IntegrationTestHelper from "../util/integration_test_helper"
+import { getCommentReplyInitialValue, replyToCommentKey } from "./CommentForms"
+import { makeComment } from "../factories/comments"
+import { makePost } from "../factories/posts"
+import { shouldIf } from "../lib/test_utils"
+import * as utilFuncs from "../lib/util"
+
+describe("ReplyButton", () => {
+  let helper, beginEditingStub, comment
+
+  beforeEach(() => {
+    helper = new IntegrationTestHelper()
+    beginEditingStub = helper.sandbox.stub()
+    comment = makeComment(makePost())
+  })
+
+  afterEach(() => {
+    helper.cleanup()
+  })
+
+  const renderButton = () =>
+    mount(
+      <Router store={helper.store} history={helper.browserHistory}>
```

Comment:
wondering about the `Router` here too

Reply:
oh, because of the `<Link>` in the `LoginPopupHelper`? you may be able to avoid this by using `shallow` instead of `mount` here